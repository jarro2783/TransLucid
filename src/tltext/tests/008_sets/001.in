var x [0 : 1..10 ∪ 20..30] = #!0;;

var y [0 : 1..10 ∪ 9..21 ∪ 20..30] = #!0;;

var z [0 : 1..10 ∪ 9..21 ∪ 20..30] = #!0;;
var z [0 : 13..15] = #!0 * 2;;

var a [0 : 1..10 ∪ 9..21 ∪ 20..30] = #!0;;
var a [0 : 18..22] = #!0 * 2;;

%%

x @ [0 <- 1];;
x @ [0 <- 2];;
x @ [0 <- 3];;
x @ [0 <- 20];;
x @ [0 <- 21];;
x @ [0 <- 22];;

y @ [0 <- 1];;
y @ [0 <- 2];;
y @ [0 <- 3];;
y @ [0 <- 4];;
y @ [0 <- 13];;
y @ [0 <- 14];;
y @ [0 <- 15];;
y @ [0 <- 19];;
y @ [0 <- 20];;
y @ [0 <- 21];;
y @ [0 <- 23];;
y @ [0 <- 24];;
y @ [0 <- 25];;

z @ [0 <- 1];;
z @ [0 <- 2];;
z @ [0 <- 3];;
z @ [0 <- 12];;
z @ [0 <- 13];;
z @ [0 <- 14];;
z @ [0 <- 15];;
z @ [0 <- 16];;
z @ [0 <- 17];;
z @ [0 <- 18];;
z @ [0 <- 25];;

a @ [0 <- 1];;
a @ [0 <- 2];;
a @ [0 <- 3];;
a @ [0 <- 18];;
a @ [0 <- 19];;
a @ [0 <- 20];;
a @ [0 <- 21];;
a @ [0 <- 22];;
a @ [0 <- 23];;
