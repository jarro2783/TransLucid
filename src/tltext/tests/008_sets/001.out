1
2
3
20
21
22
1
2
3
4
13
14
15
19
20
21
23
24
25
1
2
3
12
26
28
30
16
17
18
25
1
2
3
36
38
40
42
44
23
